### PR TITLE
use the FNV 128a from the standard library

### PR DESCRIPTION
**Pull request by marten-seemann**
It doesn't really make our code easier, but at least we can get rid of a dependency.

### CODE REVIEW

File: internal/crypto/null_aead_fnv128a.go [hunk 1]
```diff
@@ -21,21 +22,21 @@ func (n *nullAEADFNV128a) Open(dst, src []byte, packetNumber protocol.PacketNumb
...
 	if n.perspective == protocol.PerspectiveServer {
 		hash.Write([]byte("Client"))
 	} else {
 		hash.Write([]byte("Server"))
 	}
-	testHigh, testLow := hash.Sum128()
+	sum := make([]byte, 0, 16)
+	hash.Sum(sum)
+	sum = sum[:16]
+	reverse(sum)
```
Comment (lucas-clemente) [inline internal/crypto/null_aead_fnv128a.go]: Can you explain / document why this reverse is necessary?

File: internal/crypto/null_aead_fnv128a.go [hunk 2]
```diff
@@ -21,21 +22,21 @@ func (n *nullAEADFNV128a) Open(dst, src []byte, packetNumber protocol.PacketNumb
...
 	hash.Write(src[12:])
 	if n.perspective == protocol.PerspectiveServer {
 		hash.Write([]byte("Client"))
 	} else {
 		hash.Write([]byte("Server"))
 	}
-	testHigh, testLow := hash.Sum128()
+	sum := make([]byte, 0, 16)
+	hash.Sum(sum)
+	sum = sum[:16]
```
Comment (lucas-clemente) [inline internal/crypto/null_aead_fnv128a.go]: This should be unnecessary.

File: internal/crypto/null_aead_fnv128a.go (hunk 1 above)
Comment (marten-seemann) [reply internal/crypto/null_aead_fnv128a.go]: The hash is written in little endian, so we have to reverse the slice.

File: internal/crypto/null_aead_fnv128a.go (hunk 2 above)
Comment (marten-seemann) [reply internal/crypto/null_aead_fnv128a.go]: It's not if we use the return value. We should do that.

**Pull request merged**